src/progressbar/progressbar.js

```diff
@@ -10,7 +10,10 @@ angular.module('ui.bootstrap.progressbar', [])
         animate = angular.isDefined($attrs.animate) ? $scope.$parent.$eval($attrs.animate) : progressConfig.animate;
 
     this.bars = [];
-    $scope.max = angular.isDefined($scope.max) ? $scope.max : progressConfig.max;
+
+    if (!angular.isDefined($attrs.max)) {
```

Comment:
Should be able to just do `if ($attrs.max) {` since the only way it can be falsey (but defined) is if it's set via `max=""` and that's invalid.